Aruco unit tests do not compile under OpenCV 4.2 (noetic)

```
/home/rohan/noetic_catkin/src/fiducials/aruco_detect/test/aruco_images_test.cpp: In member function ‘void ArucoImagesTest::publish_image(std::string)’:
/home/rohan/noetic_catkin/src/fiducials/aruco_detect/test/aruco_images_test.cpp:44:54: error: ‘CV_LOAD_IMAGE_COLOR’ was not declared in this scope
   44 |     cv::Mat image = cv::imread(image_directory+file, CV_LOAD_IMAGE_COLOR);

```

--- a/aruco_detect/test/aruco_images_test.cpp
+++ b/aruco_detect/test/aruco_images_test.cpp
@@ -41,7 +41,7 @@ protected:
   virtual void TearDown() { delete it;}
 
   void publish_image(std::string file) {
-    cv::Mat image = cv::imread(image_directory+file, CV_LOAD_IMAGE_COLOR);
+    cv::Mat image = cv::imread(image_directory+file, cv::IMREAD_COLOR);
     cv::waitKey(30);
     sensor_msgs::ImagePtr msg = cv_bridge::CvImage(std_msgs::Header(), "bgr8", image).toImageMsg();
     image_pub.publish(msg);

